library/psa_crypto.c

```diff
@@ -2325,34 +2325,45 @@ psa_status_t psa_import_key( const psa_key_attributes_t *attributes,
     if( status != PSA_SUCCESS )
         goto exit;
 
-#if defined(MBEDTLS_PSA_CRYPTO_SE_C)
-    if( driver != NULL )
+    if( psa_key_lifetime_is_external( psa_get_key_lifetime( attributes ) ) )
```

Comment:
I'd prefer to do it the other way around:
```c
#if defined(MBEDTLS_PSA_CRYPTO_SE_C)
if( driver != NULL )
{
...
}
else
#endif
if( psa_key_lifetime_is_external( psa_get_key_lifetime( attributes ) ) )
```
That way the code is less indented and the dynamic SE code is more on its own.

Reply:
Done.